# genomewide_aligned_blocks
This repository contains the Python scripts required to generate blocks of aligned nucleotides across multiple species. Optionally, phastCons conservation scores can then be calculated for each nucleotide in an aligned block.

# ARCHIVING (May 2020)
I am archiving this repository as it requires scripts that were present in the Michigan State University HPC environment that are no longer available. The commands for the LASTZ/MULTIZ framework can still be pulled out from the codes saved here.
